File: samples/client/3_1_0_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/BooleanTypeMatchesBooleans.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.BooleanJsonSchema;

public class BooleanTypeMatchesBooleans {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class BooleanTypeMatchesBooleans1 extends BooleanJsonSchema {
        private static @Nullable BooleanTypeMatchesBooleans1 instance = null;
        public static BooleanTypeMatchesBooleans1 getInstance() {
            if (instance == null) {
                instance = new BooleanTypeMatchesBooleans1();
            }
            return instance;
        }
    }

}

File: samples/client/3_1_0_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/BooleanTypeMatchesBooleans.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.BooleanJsonSchema

class BooleanTypeMatchesBooleans: BooleanJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class BooleanTypeMatchesBooleans1 : BooleanJsonSchema.BooleanJsonSchema1() {
        companion object {
            @Volatile
            private var instance: BooleanTypeMatchesBooleans1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: BooleanTypeMatchesBooleans1().also { instance = it }
                }
        }
    }

}
